umap/static/umap/vendors/leaflet/leaflet.css

```diff
@@ -481,7 +481,7 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	}
 .leaflet-popup-content-wrapper {
 	padding: 1px;
-	text-align: left;

```

Comment:
This is not possible to modify CSS from the `vendors` folder, is that critical to left it as is?

Reply:
'text-align:left' makes content of popup always left-aligned. But it actually depends on user content not the website language. Writing system of user content can be same as UI or different from it.

'text-align:start' also isn't always correct, since with this text-alignment is obtained using direction of UI (dir=rtl/ltr). 

So I suggest to ignore this change and instead we have a hint for users about description field of popup, so they know that it's possible to control direction and alignment of popup content by wrapping it in `<div dir="rtl/ltr" style="text-align:right/left"></div>`. It can also be a button to apply this.

Reply:
I'll create an issue for this, so for now we leave it as is.